pontoon/base/signals.py

```diff
@@ -17,3 +21,71 @@ def project_locale_added(sender, **kwargs):
         project = project_locale.project
         project.has_changed = True
         project.save(update_fields=['has_changed'])
+
+
+@receiver(pre_save, sender=Locale)
+def create_locale_translators_group(sender, **kwargs):
+    """
+    Creates respective translators group in django's permission system.
+    Managers should be able to translate and manage locale.
+    """
+    instance = kwargs['instance']
+
+    if kwargs['raw'] and getattr(instance, 'translators_group', False):
```

Comment:
Why `getattr`? The moment this code starts existing so will the model attribute. You can just check if it's `None`, right?